test/XRP/issued-currency-client.test.ts

```diff
@@ -583,4 +584,77 @@ describe('Issued Currency Client', function (): void {
       assert.deepEqual(error, FakeXRPNetworkClientResponses.defaultError)
     })
   })
+
+  it('sendIssuedCurrency - success', async function (): Promise<void> {
+    // GIVEN an IssuedCurrencyClient with mocked networking that will return a successful hash for submitTransaction
+    const issuedCurrencyClient = new IssuedCurrencyClient(
+      fakeSucceedingGrpcClient,
+      fakeSucceedingJsonClient,
+      XrplNetwork.Test,
+    )
+
+    // WHEN sendIssuedCurrency is called
+    const transactionResult = await issuedCurrencyClient.sendIssuedCurrency(
+      this.wallet,
+      testAddress,
+      'FOO',
+      testAddress,
+      '100',
+      0.5,
+    )
+
+    // THEN the result is a transaction result with the expected hash
+    const expectedTransactionHash = Utils.toHex(
+      FakeXRPNetworkClientResponses.defaultSubmitTransactionResponse().getHash_asU8(),
+    )
+
+    assert.exists(transactionResult)
+    assert.equal(transactionResult.hash, expectedTransactionHash)
+  })
+
+  it('sendIssuedCurrency - classic destination address', function (): void {
+    // GIVEN an IssuedCurrencyClient with mocked networking that will return a successful hash for submitTransaction
+    const issuedCurrencyClient = new IssuedCurrencyClient(
+      fakeSucceedingGrpcClient,
+      fakeSucceedingJsonClient,
+      XrplNetwork.Test,
+    )
+
+    // WHEN sendIssuedCurrency is called with a classic address argument for destination THEN an error is thrown.
+    issuedCurrencyClient
+      .sendIssuedCurrency(
+        this.wallet,
+        testClassicAddress,
+        'FOO',
+        testAddress,
+        '100',
+        0.5,
+      )
+      .catch((error) => {
+        assert.equal(error.errorType, XrpErrorType.XAddressRequired)
+      })
```

Comment:
can we use `try/catch` here instead of a promise?

Reply:
mm yep will do that in all of these tests